_javascript/modules/components/toc.js

```diff
@@ -1,5 +1,5 @@
 export function toc() {
-  if (document.querySelector('main h2')) {
+  if (document.querySelector('main h2, h3')) {
```

Comment:
The selector needs to be improved:

```suggestion
  if (document.querySelector('main h2, main h3')) {
```